src/Mail/exceptions.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (http://nette.org)
+ * Copyright (c) 2004 David Grudl (http://davidgrudl.com)
+ */
+
+namespace Nette\Mail;
+
+use Nette;
+
+
+/**
+ * Send failed exception.
+ *
+ * @author Jan Dvořák
+ */
+class SendFailedException extends Nette\InvalidStateException
+{
+}
+
+
+/**
+ * SMTP mailer exception.
+ *
+ * @author     David Grudl
+ */
+class SmtpException extends SendFailedException
```

Comment:
Ale pak to nemůžeš poslat v SendMailMaileru a tím to ztrácí smysl, myslel jsem, že bych v SmtpMaileru dal taky tu vyjímku SendFailed ale bude to BC break